### DISCUSSION TITLE
How to apply linear gradient on a container.

### DISCUSSION THREAD

**Discussion by Haadi-A** [flet-dev/flet#937] [Q&A] [answered]
Hi @ndonkoHenri  Is there a way to apply a linear gradient to a container, I tried using ft.LinearGradient() but nothing is displayed, here is the code :

```
import flet as ft


def ProfilePage(page=ft.Page):
    backButton: ft.Control = ft.FilledButton(
        text="back", icon=ft.icons.ARROW_BACK_SHARP)
    page.add(
        ft.Container(content=ft.Column(
            controls=[backButton,
                      ft.Container(
                            height=610, width=700,
                            bgcolor=ft.LinearGradient(
                                colors=[ft.colors.BLUE, ft.colors.CYAN],
                                stops=[100.5, 100.00]
                                ), 
                                border_radius=100,
                                       ),
                                    ],
                                ),                  
            alignment=ft.alignment.center,
         ),
        )

ft.app(target=ProfilePage)

```

Plus this is what I am trying to achieve : 

![profile_page_container](https://user-images.githubusercontent.com/77079200/215229994-d80b9ad0-917c-4465-bfa4-f432dc42873e.png)

But here is what is shown:
![Screenshot from 2023-01-28 00-32-29](https://user-images.githubusercontent.com/77079200/215230182-16b05fb9-d411-42c8-8169-42ed8962036a.png)

**Comment by ndonkoHenri** [comment, marked as answer]
`bgcolor` takes only strings. You have to use the `gradient` property of the `container`. See docs here: https://flet.dev/docs/controls/container#gradient
So, modify your code as follows:
```python
ft.Container(
    height=610, width=700,
    border_radius=100,
    gradient=ft.LinearGradient(
        colors=[ft.colors.BLUE, ft.colors.CYAN],
        stops=[100.5, 100.00]
    )
)
```

**Reply by ndonkoHenri** [reply]
To better choose your gradients, I suggest you this -> https://flutils.fly.dev
A simple tool to ease your life as a Flet developer.
